src/nwbuild.js

```diff
@@ -139,3 +184,5 @@ export const nwbuild = async (options) => {
     return error;
   }
 };
+
+export default nwbuild;
```

Comment:
If you export like this, you'll need to update all of your documentation and change every occurrence of:
```js
import { nwbuild } from "nw-builder";
```
to:
```js
import nwbuild from "nw-builder";
```

This probably the better option, but you can also do this instead:

```js
export const nwbuild = async (options) => { ... };
export default nwbuild;
```

This allows either import method to work.

Reply:
```javascript
export const nwbuild = async (options) => { ... };
export default nwbuild;
```

Already made that change in the code, just update the docs too